tests/conftest.py

```diff
@@ -280,7 +285,15 @@ def microvm_factory(fc_tmp_path, bin_cloner_path):
     One can comment the removal line, if it helps with debugging.
     """
 
-    uvm_factory = MicroVMFactory(fc_tmp_path, bin_cloner_path)
+    if binary_dir := request.config.getoption("--binary-dir"):
+        uvm_factory = MicroVMFactory(
+            fc_tmp_path,
+            bin_cloner_path,
+            fc_binary_path=Path(binary_dir) / "firecracker",
+            jailer_binary_path=Path(binary_dir) / "jailer",
+        )
+    else:
+        uvm_factory = MicroVMFactory(fc_tmp_path, bin_cloner_path)
```

Comment:
We could go one step farther and pull the get_binaries call from the Microvm class all the way here, so Microvm always gets passed a firecracker binary. I have a partial change here: https://github.com/firecracker-microvm/firecracker/commit/7c33ff9259ace2ab38bffc5941b226212a8188b0

At your discretion. I can just rebase on top of this change.

Reply:
Mh, yeah, yours is a bit cleaner, so will incorporate :)

Reply:
Done :)